fiftyone/migrations/revisions/admin/v0_7_1.py

```diff
@@ -8,12 +8,8 @@
 
 
 def up(db):
-    db.admin.command({"getParameter": 1, "featureCompatibilityVersion": 1})
     db.admin.command({"setFeatureCompatibilityVersion": "4.4"})
-    db.admin.command({"getParameter": 1, "featureCompatibilityVersion": 1})
 
 
 def down(db):
-    db.admin.command({"getParameter": 1, "featureCompatibilityVersion": 1})
-    db.admin.command({"setFeatureCompatibilityVersion": "3.6"})
-    db.admin.command({"getParameter": 1, "featureCompatibilityVersion": 1})
+    db.admin.command({"setFeatureCompatibilityVersion": "4.2"})
```

Comment:
I think this should be 4.2, not 3.6, because fiftyone<0.7 was always using MongoDB 4.2. I don't recall specifics, but I think we may be using some aggregation features that were introduced in 4.2...